crates/hir/src/lib.rs

```diff
@@ -3138,6 +3135,19 @@ impl ConstParam {
     pub fn ty(self, db: &dyn HirDatabase) -> Type {
         Type::new(db, self.id.parent(), db.const_param_ty(self.id))
     }
+
+    pub fn default(self, db: &dyn HirDatabase) -> Option<String> {
+        let arg = generic_arg_from_param(db, self.id.into())?;
+        known_const_to_string(arg.constant(Interner)?, db)
```

Comment:
I considered using the old `is_unknown` method here, but it's expected to be used for concrete constant values only
https://github.com/rust-lang/rust-analyzer/blob/master/crates/hir-ty/src/consteval.rs#L31-L49